internal/usage/consts.go

```diff
@@ -17,4 +17,6 @@ const (
 	DiskSelectionUsage string = "Disk Selection"
 	//user networkType selection
 	NetworkTypeSelectionUsage string = "NetworkType"
+	//usage of platform provider other than baremetal
+	PlatformSelectionUsage string = "Platform selection"
```

Comment:
Please change it to platform type
```suggestion
	PlatformTypeUsage string = "Platform type"
```

Reply:
@gamli75  This is not just platform type. It for changing the platform (platform type different than baremetal + has credentials). I thought on "platform type" but it's not that accurate. WDYT?

Reply:
+1 for Platform selection

Reply:
agree